Reply/app/src/main/java/com/example/reply/ui/ReplyApp.kt

```diff
@@ -49,33 +61,178 @@ fun ReplyApp(
         }
     }
 
+    val drawerState = rememberDrawerState(initialValue = DrawerValue.Closed)
+    val scope = rememberCoroutineScope()
+
     if (navigationType == ReplyNavigationType.PERMANENT_NAVIGATION_DRAWER) {
-        PermanentNavigationDrawer(drawerContent = {}) {
-            ReplyAppContent(navigationType, contentType)
+        PermanentNavigationDrawer(drawerContent = {
+            NavigationDrawerContent()
+        }) {
+            ReplyAppContent(navigationType, contentType, replyHomeUIState)
         }
     } else {
-        ModalNavigationDrawer(drawerContent = {}) {
-            ReplyAppContent(navigationType, contentType)
+        ModalNavigationDrawer(
+            drawerContent = {
+                NavigationDrawerContent {
+                    scope.launch {
```

Comment:
I would prefer we use the named `onDrawerClicked` param here instead of a trailing lambda. I think trailing lambda is better used for `content`. Same below

Reply:
Got it 👍